src/ring_theory/localization/basic.lean

```diff
@@ -793,6 +793,25 @@ instance : comm_semiring (localization M) :=
   right_distrib  := λ m n k, localization.induction_on₃ m n k (by tac),
   .. localization.comm_monoid_with_zero M }
 
+/--For any given denominator `b : M`, the map `a ↦ a / b` is an `add_monoid_hom` from `R` to
+  `localization M`-/
+def mk_add_monoid_hom (b : M) : R →+ localization M :=
+{ to_fun := flip mk b,
+  map_zero' := mk_zero _,
+  map_add' := λ x y, (add_mk_self _ _ _).symm }
+
+lemma mk_finset_sum {ι : Type*} (f : ι → R) (s : finset ι) (b : M) :
```

Comment:
I think it's fine to drop the `finset` in this name if you want to; we're not all that consistent about whether to include it, and if any sum gets that privilege it's the finset one.